docs/src/main/sphinx/sql/drop-catalog.md

```diff
@@ -0,0 +1,37 @@
+# DROP CATALOG
+
+## Synopsis
+
+```text
+DROP CATALOG catalog_name
+```
+
+## Description
+
+Drops an existing catalog. Dropping a catalog does not interrupt any running
+queries that use it, but makes it unavailable to any new queries.
+
+:::{warning}
+Some connectors are known not to release all resources when dropping a catalog
+that uses such connector. This includes all connectors that can read data from
+HDFS, which are {doc}`/connector/hive`, {doc}`/connector/iceberg`,
+{doc}`/connector/delta-lake`, and {doc}`/connector/hudi`.
+:::
+
+:::{note}
+This command requires [catalog management type](/admin/properties-catalog) to
+be set to `dynamic`.
+:::
+
```

Comment:
Does it need write access to the file system? How does it work if not ..e .g. with k8s?

Reply:
In the Trino Docker image, the `/etc/trino/catalog` dir is already writeable by the user running the Trino server. I explained all the permission details in the properties page.